src/utils/setupLogic/setupHelper.js

```diff
@@ -18,12 +18,12 @@ export async function addResearchStation(gameState, cityName) {
 }
 
 //move the document to a collection
-function addTo(gameState, cityRef, cityData, collection) {
+export function addTo(gameState, cityRef, cityData, collection) {
```

Comment:
Should we move these functions somewhere... so we can use them? I think I saw the same for placing a research station and other things.